### ISSUE TITLE
Docker container

### ISSUE THREAD

**Issue by wstrange** [mkimuram/k8sviz#1]
Very cool project!

Suggestion: wrap up the dependencies as a docker container, so someone can run it via `docker run`

**Comment by mkimuram** [comment]
@wstrange  

Thank you for your suggestion.

I've created Dockerfile below and confirmed that it works inside a container.
It needs a bit long arguments to pass k8s configurations and directory to write results to.

Do you think that it still help your use case or do you have any good ideas to improve it?

```
# cat Dockerfile 
FROM alpine:3.11
RUN apk add --no-cache bash graphviz ttf-linux-libertine

ADD https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl /usr/local/bin/
RUN chmod +x /usr/local/bin/kubectl

COPY icons /icons
COPY k8sviz.sh /

CMD /k8sviz.sh

# docker build -t k8sviz:0.1 .

# docker run --network host -v ~/work:/work -v ~/kubeconfig:/config:ro -e KUBECONFIG=/config -it k8sviz:0.1 /k8sviz.sh -n default -t png -o /work/default.png 
```

**Comment by mkimuram** [comment]
@wstrange

FYI: Implemented go version in #4 .

It only depends on dot (graphviz) command, so we don't need to worry so much on dependencies, now. (Still considering a better way to containerize this, though.)
